packages/flutter/lib/src/material/drawer.dart

```diff
@@ -228,6 +229,17 @@ class DrawerController extends StatefulWidget {
   /// By default, the color used is [Colors.black54]
   final Color scrimColor;
 
+  /// The horizontal width from the edge of the screen that will respond to a
```

Comment:
Maybe: 

The width of the area within which a horizontal swipe will open the drawer is the larger of this value and the corresponding [MediaQuery] padding inset.